src/Google.Api.CommonProtos/Type/MoneyPartial.cs

```diff
@@ -0,0 +1,47 @@
+﻿/*
+ * Copyright 2018 Google Inc. All Rights Reserved.
+ * Use of this source code is governed by a BSD-style
+ * license that can be found in the LICENSE file or at
+ * https://developers.google.com/open-source/licenses/bsd
+ */
+
+using System;
+
+namespace Google.Type
+{
+    public partial class Money
+    {
+        /// <summary>
+        /// The amount of money in <see cref="decimal"/> format.

```

Comment:
It would be worth explaining that this is a view over `Units` and `Nanos`: getting this property combines those property values and setting this property will set both of those properties.